content/posts/2022-features.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "2022 for Packit"
+date: 2023-01-28T11:58:50+01:00
```

Comment:
You should probably add a weight to this document, because now it's at the bottom of the blog posts lists and IMO quite hard to find (or even spot that there is something new)

Reply:
I hope it's just a joke..;)